src/translator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+responsible for translating the article title into 'owo speak'
+*/
+
+class translator
+{
+    /*
+    responsible for adding stutter to the article title.
+    This stutter has a 30% chance of occuring on the first word of the title
+    and voids the chance if the first word of the title starts with a illigal character ie " ' " or ' " ' .
+    */
+    public function stutter($str)
+    {
+        if (mt_rand(1, 100) <= 30) {
+            $split = explode(" ", $str);
+            $firstLetter = substr($split[0], 0, 1);
+
+            if (!preg_match("/[a-zA-Z]/", $firstLetter)) {
+                return $str;
+            }
+
+            $generate = join(" ", $split);
+            $str = "$firstLetter-$firstLetter-$generate";
+        }
+        return $str;
+    }
+
+    /*
+    responsible for translating the article title into 'uwu speak'.
+    This is simpely done by replacing all occurences of 'r' and 'l' with a 'w' respectively.
+    */
+    public function translateToUWU($str)
+    {
+        $pattern = '/[lr]/i';
+        $str = preg_replace($pattern, 'w', $str);
+
+        return $str;
+    }
+
+    /* 
+    responsible for selecting a random 'emote' and 'owo' from the content.json file found in ./assets/. 
+    */
+    private function randomFromArray(array $inputArray)
+    {
+        if (gettype($inputArray) != "array") {
+            die("Given var in JSON is not an array");
```

Comment:
Are you sure you want the whole program to stop at this point instead of throwing an exception that's eventually catched and logged within `run.php`?

Reply:
Thanks for the heads up! commit 0ffd9ec459fdcfae3425c46ade624d2109ffc791